gulpfile.babel.js

```diff
@@ -120,75 +103,81 @@ gulp.task('copy:license', () =>
 gulp.task('copy:main.css', () => {
   const banner = `/*! HTML5 Boilerplate v${pkg.version} | ${pkg.license} License | ${pkg.homepage} */\n\n`;
 
-  gulp.src(`node_modules/main.css/dist/main.css`)
+  return gulp.src('node_modules/main.css/dist/main.css')
     .pipe(plugins().header(banner))
     .pipe(plugins().autoprefixer({
-      browsers: ['last 2 versions', 'ie >= 9', '> 1%'],
       cascade: false
     }))
     .pipe(gulp.dest(`${dirs.dist}/css`));
 });
 
 gulp.task('copy:misc', () =>
   gulp.src([
-
     // Copy all files
     `${dirs.src}/**/*`,
 
     // Exclude the following files
     // (other tasks will handle the copying of these files)
     `!${dirs.src}/css/main.css`,
     `!${dirs.src}/index.html`
-
   ], {
-
-    // Include hidden files by default
-    dot: true
-
-  }).pipe(gulp.dest(dirs.dist))
+      // Include hidden files by default
```

Comment:
minor nitpick here - but the indentation is a little bit off. should be 4 spaces (not 6) on lines 124 and 125 and indent of 2 spaces on line 126.

Reply:
fixed